tdrs-backend/tdpservice/parsers/row_schema.py

```diff
@@ -45,6 +48,7 @@ def parse_and_validate(self, line, generate_error):
         if not preparsing_is_valid:
             if self.quiet_preparser_errors:
                 return None, True, []
+            logger.info(f"Preparser error encountered. Errors: {preparsing_errors}.")
```

Comment:
logging the actual error here may potentially be noisy, especially since all the errors are logged at the end of `parser_task`